presto-docs/src/main/sphinx/release/release-0.153.rst

```diff
@@ -6,32 +6,154 @@ General Changes
+* Add :func:`typeof` for discovering expression types.
+* Add decimal support to :func:`avg`, :func:`ceil`, :func:`floor`, :func:`round`,
+  :func:`truncate`, :func:`abs`, :func:`mod` and :func:`sign`.
+* Add :func:`shuffle` function for arrays.
 * Add :doc:`/connector/accumulo`.
 
-SPI Changes
+Web UI Changes
+--------------
+
+* Fix rendering failures in due to null nested data structures.
+* Fix active worker count on cluster overview page to not include coordinator.
+* Replace buffer skew indicators on query details page with scheduled time skew.
+* Add stage total buffer, pending tasks and wall time to stage statistics on query details page.
+* Add option to filter task lists by status on query details page.
+* Add copy button for query text, query ID, and user to query details page.
+
+JDBC Driver Changes
+-------------------
+
+* Add support for ``REAL`` data type which corresponds to the Java ``float`` type.
+
+CLI Changes
 -----------
 
-* Add support for pluggable resource group management. A ``Plugin`` can now
-  provide management factories via ``getResourceGroupConfigurationManagerFactories()``
-  and the factory can be enabled via the ``etc/resource-groups.properties``
-  configuration file by setting the ``resource-groups.configuration-manager``
-  property. See the ``presto-resource-group-managers`` plugin for an example.
+* Add support for configuring the HTTP Truststore.
 
 Hive Changes
 ------------
 
+* Fix permissions for new tables when using SQL-standard authorization.
+* Allow certain combinations of queries to be executed in a transaction-ish manner,
+  for example, when dropping a partition and then recreating it. Atomicity is not
+  guaranteed due to fundamental limitations in the design of Hive.
+* Fail queries that attempt to rename partition columns.
+* Add support for ORC bloom filters in predicate push down.
+  This is can be enabled using the ``hive.orc.bloom-filters.enabled``
+  configuration property or the ``orc_bloom_filters_enabled`` session property.
+* Add new optimized RCFile reader.
+  This can be enabled using the ``hive.rcfile-optimized-reader.enabled``
+  configuration property or the ``rcfile_optimized_reader_enabled`` session property.
+* Add support for the Presto ``REAL`` type which corresponds to the Hive ``FLOAT`` type.
 * Add support for ``char(x)`` type.
+* Add support for creating, dropping and renaming schemas (databases).
+  The filesystem location can be specified when creating a schema,
+  which allows, for example, easily creating tables on S3.
+* Record Presto query ID for tables or partitions written by Presto
+  using the ``presto_query_id`` table or partition property.
+* Support per-transaction cache for Hive metastore.
+* Include path name in error message when listing a directory fails.
+* Rename ``allow-all`` authorization method to ``legacy``. This
+  method is deprecated and will be removed in a future release.
+* Do not retry intentionally-aborted S3 requests.
```

Comment:
"intentionally-aborted" sounds weird. I'd rephrase it as "Do not retry S3 requests that are aborted intentionally".